doc/src/stdlib/iterators.md

```diff
@@ -1,15 +1,17 @@
 # Iteration utilities
 
 ```@docs
-Base.Iterators.zip
-Base.Iterators.enumerate
-Base.Iterators.rest
 Base.Iterators.countfrom
-Base.Iterators.take
-Base.Iterators.drop
 Base.Iterators.cycle
-Base.Iterators.repeated
-Base.Iterators.product
+Base.Iterators.drop
+Base.Iterators.enumerate
+Base.Iterators.filter
 Base.Iterators.flatten
 Base.Iterators.partition
+Base.Iterators.product
+Base.Iterators.repeated
+Base.Iterators.rest
+Base.Iterators.reverse
+Base.Iterators.take
+Base.Iterators.zip
```

Comment:
Note: added `reverse` and `filter` and sorted alphabetically.

Reply:
Hmm, I think I may want to undo the sorting... now that I look at it, there was some sense to the previous ordering.